src/xenia/xbox.h

```diff
@@ -547,6 +547,29 @@ enum class XDeploymentType : uint32_t {
   kUnknown = 0xFF,
 };
 
+inline bool IsOfflineXUID(uint64_t xuid) { return ((xuid >> 60) & 0xF) == 0xE; }
+
+inline bool IsOnlineXUID(uint64_t xuid) {
+  return ((xuid >> 48) & 0xFFFF) == 0x9;
+}
+
+inline bool IsGuestXUID(uint64_t xuid) {
+  uint32_t HighPart;
+  HighPart = (xuid >> 48);
```

Comment:
Pit these two lines into one also you can add const. Plus there is no need to add () in this case as there is only one operation